### DISCUSSION TITLE
Upgrade path from self hosted Trilium setup currently running 0.63.7

### DISCUSSION THREAD

**Discussion by obuno** [TriliumNext/Notes#362] [Q&A / Support] [answered]
Hi there team,

Is there anything special to take care about if I'm planning to "upgrade" from that latest stock version of Trilium (0.63.7) ?
I'd consider any possible paths, fresh LXC container, TriliumNext setup + import DB is fine, just need some guidance around what is possible. Thanks !

Let me know,
Kind regards,
o.

**Comment by meichthys** [comment, marked as answer]
Nothing special, just update your docker compose to the new image, or install the TriliumNext app. The new app will just use the same database as the old app. If you run into any troubles,  you can export your notes from Trilium and import into TriliumNext.

**Reply by eliandoran** [reply]
@obuno , additionally to @meichthys 's answer, you can find the updated docker-compose file here: https://github.com/TriliumNext/Notes/blob/develop/docker-compose.yml

Generally you can reuse the same data directory without any changes and there will be no database migration needed.

Let us know if you come across any issues.

**Comment by obuno** [comment]
Hi team, upgraded and updated `ln -nsf $(pwd)/trilium-0-90-4 $(pwd)/trilium` and everything seems working fine. for the record, absolutely no Docker of any forms here. thanks
